interoperability_report.py

```diff
@@ -673,21 +708,24 @@ def main():
                     elif len(v) == 2:
                         check_function = no_check
                     else:
-                        print('Error in the definition of the Test Suite. Number of arguments incorrect.')
+                        print('Error in the definition of the Test Suite. \
+                                Number of arguments incorrect.')
                         break
                     assert(len(parameters) == len(expected_codes))
                     if (options['test_cases'] == None or k in options['test_cases']) \
-                        and (options['test_cases_disabled'] == None or k not in options['test_cases_disabled']):
+                        and \
+                       (options['test_cases_disabled'] == None
+                            or k not in options['test_cases_disabled']):
```

Comment:
```suggestion
                        and (options['test_cases_disabled'] == None
                            or k not in options['test_cases_disabled']):
```

Reply:
I think that we can delete the functions `check_disable_test` and `check_test_case_in_test_suite`. Here we should split the condition
```
if (options['test_cases'] is not None and k not in options['test_cases']):
    #print that the test doesn't exist
    continue;
elif (options['test_cases_disabled'] is not None and k in options['test_cases_disabled']):
    #print that the test is not being run (with verbosity)
    continue;
else: # this should be the case where the test (if test_cases has values or test_cases = None) exists, or it is not in the test_cases_disabled
    # run the test
```

Reply:
done